internal/app/command/create_or_update_port.go

```diff
@@ -24,12 +24,12 @@ func NewCreateOrUpdatePortHandler(store CreateOrUpdatePortStore) *CreateOrUpdate
 }
 
 type CreateOrUpdatePortStore interface {
-	CreateOrUpdatePort(ctx context.Context, port *store.Port) error
+	CreateOrUpdatePort(ctx context.Context, port *repository.Port) error
```

Comment:
I don' understand why the `command` package uses a repository model? This by pass completely the core so tightly couple two packages outside core that should be swappable...
Assume that `repository.Port` is specific of a DB technology (sqlite, mongo, gRPC, etc) in fact this would be a good case for subpackages (/internal/repository/port/postgres/db.go if you allow for many repositories and for many implementations), so the command would need to change if repository change. To avoid this we always have driver and driven code (left and right) of the hexagon to depend on core models.

Reply:
yeah, that's what happens when you work on multiple codebase at once :) and show something not yet completed. I'll fix it, cheers

Reply:
I've pushed the changes. More details [here](https://github.com/evt/port-api/issues/2#issuecomment-1591998725).